backend/tests/conftest.py

```diff
@@ -164,6 +184,36 @@ def default_data_product(
     return data_product
 
 
+@pytest.fixture
+def admin_data_product_membership(session, admin_user):
+    data_product_membership = DataProductMembershipModel(
+        user_id=admin_user.id,
+        role=DataProductUserRole.MEMBER,
+    )
+    return data_product_membership
+
+
+@pytest.fixture
+def admin_data_product(
```

Comment:
what is the usecase of this admin data product?
To me the admin data product is just a normal data product that the admin is not the owner / member of, but should still be able to interact with. Am I missing something else?